client/flutter/lib/components/page_scaffold/page_header.dart

```diff
@@ -32,34 +35,52 @@ class PageHeader extends StatelessWidget {
     List<Widget> headerItems = [
       if (this.showBackButton)
         Transform.translate(offset: Offset(-12, 0), child: BackArrow()),
-      Expanded(child: Column(
-        mainAxisAlignment: MainAxisAlignment.center,
-        crossAxisAlignment: CrossAxisAlignment.stretch,
-        children: <Widget>[
-          //Todo: Decide what to do when title text overflow
-          AutoSizeText(this.title,
-              maxLines: 1,
-              minFontSize: 8,
-              overflow: TextOverflow.fade,
-              softWrap: false,
-              style: TextStyle(
+      Expanded(
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          crossAxisAlignment: CrossAxisAlignment.stretch,
+          children: <Widget>[
+            Semantics(
+              header: true,
+              namesRoute: true,
+              //Todo: Decide what to do when title text overflow
+              child: AutoSizeText(
+                this.title,
+                maxLines: 1,
+                minFontSize: 8,
+                overflow: TextOverflow.fade,
+                softWrap: false,
+                style: TextStyle(
                   color: Color(0xff1A458E),
                   fontWeight: FontWeight.w900,
                   fontSize: 24,
-                  letterSpacing: -0.5)),
-          SizedBox(height: 4),
-          AutoSizeText(this.subtitle,
+                  letterSpacing: -0.5,
+                ),
+              ),
+            ),
+            SizedBox(height: 4),
+            AutoSizeText(
+              this.subtitle,
               maxLines: 1,
               minFontSize: 8,
               overflow: TextOverflow.ellipsis,
               softWrap: false,
               style: TextStyle(
-                  color: Color(0xff3C4245),
-                  fontSize: 16,
-                  fontWeight: FontWeight.w700)),
-        ],
-      ),),
-      if (this.showLogo) Image.asset('assets/images/mark.png', width: 70)
+                color: Color(0xff3C4245),
+                fontSize: 16,
+                fontWeight: FontWeight.w700,
+              ),
+            ),
+          ],
+        ),
+      ),
+      if (this.showLogo)
+        Image.asset(
+          'assets/images/mark.png',
+          width: 70,
+          // TODO check if this is the right label
+          semanticLabel: 'World Health Organisation Logo',
```

Comment:
I'd say the logo doesn't need a semantic label, it's just visual decoration.

Reply:
Okay, should semantics be completely removed then, it was previously reading something like "image"

Reply:
@britannio yes, I'd say so, a good rule of thumb is "if I was forced remove this image what text would I replace it with?"... in this case you'd just leave it empty